packages/lumx-core/src/scss/components/popover/lumapps/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$lumx-popover-arrow-size: 5px !default;
+$lumx-popover-background-color: lumx-color-variant('light', 'N') !default;
+$lumx-popover-color: lumx-color-variant('dark', 'N') !default;
+$lumx-popover-elevation: 3 !default;
+$lumx-popover-font-size: map-get($lumx-typography-font-size, 'caption') !default;
+$lumx-popover-line-height: map-get($lumx-typography-line-height, 'caption') !default;
```

Comment:
Only the `$lumx-popover-arrow-size` variable is used in the popover style, the others should be removed
```suggestion
```